alf/trainers/policy_trainer.py

```diff
@@ -200,6 +242,10 @@ def _markdownify(paragraph):
             env_steps_metric = self._algorithm.get_step_metrics()[1]
             total_time_steps = env_steps_metric.result()
             iter_num += 1
+
+            self._trainer_progress._iter_num.fill_(iter_num)
+            self._trainer_progress._env_steps.fill_(total_time_steps)
```

Comment:
Update using a member function of TrainerProgress:
```
 self._trainer_progress.update(iter_num, total_time_steps)
```
update() can also do the calculation of progress() so it not calculated every time it's called.